prowler/lib/outputs/html/html.py

```diff
@@ -0,0 +1,567 @@
+import html
+import importlib
+import sys
+from os import path
+
+from prowler.config.config import (
+    html_file_suffix,
+    html_logo_url,
+    prowler_version,
+    square_logo_img,
+    timestamp,
+)
+from prowler.lib.logger import logger
+from prowler.lib.outputs.compliance.compliance import get_check_compliance
```

Comment:
This shouldn't be needed here, please get that information from the finding object.

Reply:
True